backend/src/main/java/gov/cdc/usds/simplereport/validators/CsvValidatorUtils.java

```diff
@@ -448,12 +449,31 @@ public static List<FeedbackMessage> validateDateFormat(ValueOrError input) {
   }
 
   public static List<FeedbackMessage> validateDateTime(ValueOrError input) {
-    List<FeedbackMessage> errors = new ArrayList<>(validateRegex(input, DATE_TIME_REGEX));
-    if (input.getValue() != null
-        && errors.isEmpty()
-        && input.getValue().matches(TIMEZONE_SUFFIX_REGEX)) {
-      errors.addAll(validateDateTimeZoneCode(input));
+    String value = parseString(input.getValue());
+    List<FeedbackMessage> errors = new ArrayList<>();
+
+    if (value == null) {
+      return errors;
```

Comment:
I appreciate that with this update the fact that we're potentially passing back an empty error array is visible.  I don't think that's what I would have expected based on 

```java
List<FeedbackMessage> errors = new ArrayList<>(validateRegex(input, DATE_TIME_REGEX));
```